tensorflow/java/src/main/native/BUILD

```diff
@@ -43,6 +43,7 @@ tf_cuda_library(
             "//tensorflow/core:all_kernels",
             "//tensorflow/core:direct_session",
             "//tensorflow/core:ops",
+            "//tensorflow/core/distributed_runtime/rpc:grpc_server_lib",
```

Comment:
Should we add this dependency to https://github.com/tensorflow/tensorflow/blob/ea718e872df737a6ee69f53a8f9b212c9c8197ec/tensorflow/java/BUILD#L384 instead?

Reply:
Done.